submissions/Nikita_Lemesiev/css_and_html_practice/index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&family=Silkscreen&display=swap" rel="stylesheet">
+    <title>Pop-up</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <header>
+        <div id="logo">
+           Kottans
+        </div>
+        <nav id="nav_bar">
+            <ul>
+                <li tabindex="0">Mail</li>
```

Comment:
`tabindex` is a hack for this task and it ruins your understanding of how this should work. You are forcing element behavior that should not be there at all. `li` is not an interactive element. It should not be possible to target/focus it. `li` should store a link somewhere, since it's a navigation menu. So there should be an `a` tag here, and it will be focusable by default. Please avoid using `tabindex`es. If you feel the need to use it - it means that element order is wrong.